kolibri_explore_plugin/assets/src/constants.js

```diff
@@ -18,9 +19,12 @@ export const pageNameToModuleMap = {
   [PageNames.TOPICS_ROOT]: 'topicsRoot',
   [PageNames.TOPICS_CHANNEL]: 'topicsTree',
   [PageNames.TOPICS_CONTENT]: 'topicsTree',
+  [PageNames.TOPICS_TEST]: 'topicsTree',
   [PageNames.TOPICS_TOPIC]: 'topicsTree',
 };
 
 export const CUSTOM_PRESENTATION_TITLE = 'custom-channel-ui';
 
 export const CarouselItems = 5;
+
+export const TestChannelId = 'f5f6729f95b55753badeaa066fa6e986';
```

Comment:
Instead just one test channel id we can add a button for each custom presentation that we've [here](https://github.com/endlessm/kolibri-explore-plugin/blob/master/kolibri_explore_plugin/assets/src/customApps.js#L5-L10)